### PR TITLE
About review page

### PR DESCRIPTION
**Pull request by kotalwar-krishna**
About page is added and review page is added.

about page related code you find in about.css and about.html

review page releated code you can find in review.css and review.html

in review page card component is used from my component library.
img15
img16
img17

### CODE REVIEW

File: Screens/about.css [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+}
+  
+.about .image img {
+    margin: 1rem 2rem;
+    width: 90%;
+}
+    
+.about .content span {
+    color: #bac34e;
```
Comment (psychoactiv) [inline Screens/about.css]: Try using Color Variables

Comment (shaksham08) [inline Screens/about.css]: Try to use color variables

File: Screens/about.css [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    font-size: 2.5rem;
+  }
+  
+.about .content h3 {
+    color: #222;
```
Comment (shaksham08) [inline Screens/about.css]: Same here

**Review by psychoactiv — APPROVED** [review]

**Review by shaksham08 — APPROVED** [review]
I have added one comment , u can use color variables wherever you have used hardcoded colors

File: Screens/about.css [hunk 3]
```diff
@@ -0,0 +1,28 @@
+.about {
+    display: flex;
+    margin: 4rem;
+
```
Comment (sakleprajwal) [inline Screens/about.css]: Avoid unnecessary spaces.

**Review by sakleprajwal — APPROVED** [review]
The code looks good to merge. Please go ahead.

File: Screens/about.css (hunk 3 above)
Comment (kotalwar-krishna) [reply Screens/about.css]: ok

File: Screens/about.css (hunk 1 above)
Comment (kotalwar-krishna) [reply Screens/about.css]: I miss to push origni i will do

Comment (kotalwar-krishna) [reply Screens/about.css]: I miss to push origni i will do

File: Screens/about.css (hunk 2 above)
Comment (kotalwar-krishna) [reply Screens/about.css]: yes

**Pull request merged**